src/interface.rs

```diff
@@ -151,7 +151,8 @@ where
     /// Most likely there was a mistake with the 2in9 busy connection
     /// //TODO: use the #cfg feature to make this compile the right way for the certain types
     pub(crate) fn is_busy(&self, is_busy_low: bool) -> bool {
-        (is_busy_low && self.busy.is_low()) || (!is_busy_low && self.busy.is_high())
+        (is_busy_low && self.busy.is_low().unwrap_or(false))
+            || (!is_busy_low && self.busy.is_high().unwrap_or(false))
```

Comment:
this gives different results for the devices with different is_busy_low variants. this should be changed to either return false (or true) for both cases. False might be better, because when we get an error we should better just block other stuff than maybe breaking things...